src/hooks/apiHooks/useCoupons/index.tsx

```diff
@@ -0,0 +1,22 @@
+import couponsApi from "services/api/couponsApi";
+
+function useCoupons() {
+  async function getAllCoupons() {
```

Comment:
```suggestion
  async function getCouponsList() {
```

The method name is good, but may we follow the naming conventions of the other service files, in order to prevent some confusion in the future.